test/e2e/framework/util.go

```diff
@@ -4153,7 +4153,9 @@ func CheckNodesReady(c clientset.Interface, size int, timeout time.Duration) ([]
 
 		// Filter out not-ready nodes.
 		FilterNodes(nodes, func(node v1.Node) bool {
-			return IsNodeConditionSetAsExpected(&node, v1.NodeReady, true)
+			nodeReady := IsNodeConditionSetAsExpected(&node, v1.NodeReady, true)
+			networkReady := IsNodeConditionUnset(&node, v1.NodeNetworkUnavailable) || IsNodeConditionSetAsExpectedSilent(&node, v1.NodeNetworkUnavailable, false)
```

Comment:
Any reason why it's ok for the condition being unset? Also maybe don't make the second part silent so we have some logs if some nodes aren't ready?

Reply:
This is the same set of conditions that IsNodeSchedulable checks, minus unschedulable==false. I can make the second part verbose in this case if you prefer.

Reply:
Ok.

> I can make the second part verbose in this case if you prefer.

I'd recommend that to help with debugging.

Reply:
Done